api/trace_service.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+// [#proto-status: draft]
+
+package envoy.api.v2;
+
+import "api/base.proto";
+import "trace.proto";
+
+import "google/api/annotations.proto";
+
+import "validate/validate.proto";
+
+// Service for streaming traces to server that consumes the trace data. It
+// uses OpenCensus data model as a standard to represent trace information.
+service TraceService {
+  // Envoy will connect and send StreamTracesMessage messages forever. It does
+  // not expect any response to be sent as nothing would be done in the case
+  // of failure.
+  rpc StreamTraces(stream StreamTracesMessage) returns (StreamTracesResponse) {
+  }
+}
+
+message StreamTracesResponse {
```

Comment:
StreamingTracesResponse would be a proper noun phrase.

Reply:
The names are consistent with https://github.com/envoyproxy/data-plane-api/blob/master/api/metrics_service.proto